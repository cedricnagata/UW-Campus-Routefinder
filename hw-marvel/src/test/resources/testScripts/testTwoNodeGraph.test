# test parseData and findPath on two nodes graph

# build graph
LoadGraph twoNodeGraph twoNodeGraph.csv

# check graph is built
ListNodes twoNodeGraph
ListChildren twoNodeGraph node1
ListChildren twoNodeGraph node2

# check findPath
FindPath twoNodeGraph node1 node2