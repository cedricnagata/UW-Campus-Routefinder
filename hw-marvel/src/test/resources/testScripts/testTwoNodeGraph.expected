# test parseData and findPath on two nodes graph

# build graph
loaded graph twoNodeGraph

# check graph is built
twoNodeGraph contains: node1 node2
the children of node1 in twoNodeGraph are:
the children of node2 in twoNodeGraph are:

# check findPath
path from node1 to node2:
no path found